tests/data/placebo/test_sqs_add_statements_policy/sqs.ListQueues_1.json

```diff
@@ -0,0 +1,21 @@
+{
+    "status_code": 200,
+    "data": {
+        "QueueUrls": [
```

Comment:
use a region without the extra queues, this causing lots of extraneous data recordings.

Reply:
I tried using us-west-2 and it spun up the queue as expected, but my tests still had a default to also include us-east-1. Is there something in the codebase that defaults the placebo tests to always run in us-east-1?

Reply:
on load_policy pass config={'region': 'us-west-2'}, also set your environment variables re AWS_DEFAULT_REGION=us-west-2

Reply:
Cleaned these up 👍